conans/test/unittests/client/generators/cmake_test.py

```diff
@@ -367,7 +367,8 @@ def setUp(self):
         self.conanfile.initialize(settings, EnvValues())
         ref = ConanFileReference.loads("my_pkg/0.1@lasote/stables")
         cpp_info = CppInfo("dummy_root_folder1")
-        cpp_info.name = "MyPkG"
+        cpp_info.name = "MyUnusedPkGName"
+        cpp_info.names["cmake"] = "MyCustomCMakePkgName"
```

Comment:
I think that the test should be extended for the `cmake_multi`, `cmake_find_package`, and `cmake_find_package_multi`